src/vs/workbench/contrib/terminalContrib/accessibility/browser/terminalAccessibleBuffer.ts

```diff
@@ -325,3 +363,6 @@ export class AccessibleBufferWidget extends DisposableStore {
 		return model!;
 	}
 }
+
+interface ITerminalCommandInEditor { command: ITerminalCommand; lineNumber: number }
```

Comment:
To match the function above? `ICommandWithEditorLine`? Can also omit terminal as you're not exporting